dts/bindings/pwm/ite,it8xxx2-pwm.yaml

```diff
@@ -52,6 +52,12 @@ properties:
   pinctrl-names:
     required: true
 
+  output-open-drain-enable:
```

Comment:
should this be a devicetree pwm flag instead? that way you can set it for each channel

Reply:
Because PWM driver belongs to multiple instances (e.g. pwm0, pwm1, pwm2...), we can also set it for each channel through this property.

Reply:
Sure that would work because you have a device per channel, but the PWM API semantics would imply that channel properties are in channel flags. That is if the driver was implementing multiple channels you could have something like

```
pwms = <&pwm0 0 PWM_HZ(100) PWM_POLARITY_INVERTED>,
        <&pwm0 1 PWM_HZ(100) (PWM_POLARITY_INVERTED | PWM_IT8XXX2_OPEN_DRAIN)>;
```

Reply:
Thank you for your clear explanation. It has been modified.